drivers/adc-dac/ad5592r/ad5592r-base.h

```diff
@@ -77,12 +77,31 @@ enum ad5592r_registers {
 	AD5592R_REG_RESET		= 0xF,
 };
 
+#define AD5592R_REG_PD_PD_ALL		BIT(10)
 #define AD5592R_REG_PD_EN_REF		BIT(9)
-#define AD5592R_REG_CTRL_ADC_RANGE	BIT(5)
-#define AD5592R_REG_CTRL_DAC_RANGE	BIT(4)
+#define AD5592R_REG_PD_CHANNEL(x)		BIT(x)
 
+#define AD5592R_REG_CTRL_ADC_PC_BUFF		BIT(9)
+#define AD5592R_REG_CTRL_ADC_BUFF_EN		BIT(8)
+#define AD5592R_REG_CTRL_CONFIG_LOCK		BIT(7)
+#define AD5592R_REG_CTRL_W_ALL_DACS		BIT(6)
+#define AD5592R_REG_CTRL_ADC_RANGE		BIT(5)
+#define AD5592R_REG_CTRL_DAC_RANGE		BIT(4)
+
+#define AD5592R_REG_ADC_SEQ_REP					BIT(9)
```

Comment:
You might not have known this, but we use 8 space tabs, try aligning these values to match the others around.

https://github.com/analogdevicesinc/no-OS/blob/bad1c1f2e02d2cb1bfcafc3c1c465819a842b9cd/drivers/adc-dac/ad5592r/ad5592r-base.h?ts=8